charts/trino/templates/_helpers.tpl

```diff
@@ -120,3 +120,19 @@ Code is inspired from bitnami/common
   {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the name of the file auth secret to use
+*/}}
+{{- define "trino.fileAuthSecretName" -}}
+{{- if and .Values.auth .Values.auth.passwordAuthSecret }}
+{{- .Values.auth.passwordAuthSecret | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
```

Comment:
Should we use `hasPrefix .Release.Name $name` instead of `contains`?

Reply:
See new commit. There were a number of functions that previously used contains that now use hasPrefix. This also means the if logic has changed.